backend/routes/graph.py

```diff
@@ -93,56 +98,35 @@ async def concept_graph(
     verbose and timer('concept_graph()')
 
     # Get concepts & metadata
-    concepts_raw: List[Dict] = [dict(x) for x in get_cset_members_items(
-        codeset_ids=codeset_ids, columns=['concept_id', 'vocabulary_id', 'standard_concept'])]
-    # Filter
-    # TODO: see: filter_concepts()
-    concepts: List[Dict]
+    concepts_unfiltered: List[RowMapping] = get_cset_members_items(
```

Comment:
## `concept_graph()` simplification
I did a test case just to double check that the following wer equivalent.

<details><summary>Before</summary>
<p>

```py
# Filter: by vocab & non-standard
concepts: List[Dict]
hidden_by_voc: Dict[str, Set[int]]
nonstandard_concepts_hidden: Set
concepts, hidden_by_voc, nonstandard_concepts_hidden = filter_concepts(
    concepts_raw, hide_vocabs, hide_nonstandard_concepts)

# Fill gaps
# - find missing in-between nodes
missing_in_betweens_ids: Set[int] = get_missing_in_between_nodes(REL_GRAPH, concept_ids, False)
# - filtering
missing_in_betweens: List[RowMapping] = get_concepts(missing_in_betweens_ids)
concepts_m: List[Dict]
hidden_by_voc_m: Dict[str, Set[int]]
nonstandard_concepts_hidden_m: Set
concepts_m, hidden_by_voc_m, nonstandard_concepts_hidden_m = filter_concepts(
    missing_in_betweens, hide_vocabs, hide_nonstandard_concepts)
# - combine missing in-betweens into other objects
concept_ids.update(missing_in_betweens_ids)
for voc, node_ids in hidden_by_voc.items():
    hidden_by_voc[voc] = hidden_by_voc[voc].union(hidden_by_voc_m[voc])
nonstandard_concepts_hidden = nonstandard_concepts_hidden.union(nonstandard_concepts_hidden_m)
```

</p>
</details> 

<details><summary>After</summary>
<p>

```py
# Fill gaps
missing_in_betweens_ids: Set[int] = get_missing_in_between_nodes(
    REL_GRAPH, set([c['concept_id'] for c in concepts_unfiltered]), False)
missing_in_betweens: List[RowMapping] = get_concepts(missing_in_betweens_ids)
concepts_unfiltered2: List[RowMapping] = concepts_unfiltered + missing_in_betweens

# Filter: by vocab & non-standard
concepts: List[Dict[str, Any]]
hidden_by_voc: Dict[str, Set[int]]
nonstandard_concepts_hidden: Set
concepts, hidden_by_voc, nonstandard_concepts_hidden = filter_concepts(
    concepts_unfiltered2, hide_vocabs, hide_nonstandard_concepts)
```

</p>
</details>